transcriptic/version.py

```diff
@@ -1 +1 @@
-__version__ = "9.4.2"
+__version__ = "9.4.3"
```

Comment:
this is technically a feature bump and not a bugfix, so it sohuld be 9.5.0

Also, we usually separate the release commit from the feature commit.